noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -2036,56 +2217,37 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
-        <title>Conversion count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, COUNT(*) number_of_conversions
-                    FROM arkivstruktur.konvertering k
-                    JOIN arkivstruktur.dokumentobjekt d ON k._parent_id = d._id
-                    JOIN arkivstruktur.dokumentbeskrivelse dd ON dd.systemid = d._ref_dokumentbeskrivelse
-                    JOIN arkivstruktur.registrering r ON r.systemid = dd._ref_registrering
-                    GROUP BY r._ref_arkivdel;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    <!--
+    ============================================================================================================
+    LOEPENDEJOURNAL
+    ============================================================================================================
+    -->
+    <!-- Checks -->
 
-    <rule>
-        <title>Deletion apart from disposal count</title>
-        <group>arkivstruktur</group>
+    <check id="LJC1">
+        <title>Number of screened registry entries</title>
+        <description>Number of screened Registry Entries found in loependejournal</description>
+        <group>loependejournal</group>
         <queries>
             <info>
                 <![CDATA[
-                    SELECT s.systemid series_system_id,
-                        COALESCE(series.val, 0) number_of_series_deletions_apart_from_disposals,
-                        COALESCE(document.val, 0) number_of_document_descriptions_deletions_apart_from_disposals
-                    FROM arkivstruktur.arkivdel s
-                    LEFT JOIN
-                    (
-                        SELECT sl._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.sletting sl
-                        GROUP BY sl._ref_arkivdel
-                    ) series ON series.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.sletting sl
-                        JOIN arkivstruktur.dokumentbeskrivelse d ON d.systemid = sl._ref_dokumentbeskrivelse
-                        JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
-                        GROUP BY r._ref_arkivdel
-                    ) document ON document.series_system_id = s.systemid;
+                    SELECT COUNT(tilgangsrestriksjon) screened_registry_entries
+                    FROM loependejournal.journalpost;
                 ]]>
             </info>
         </queries>
-    </rule>
+    </check>
+
+    <!-- Tests -->
 
-    <rule>
+    <test id="LJT1">
         <title>Registry Entry count</title>
+        <description>
```

Comment:
`Provides information about the number of registry entries found in loependeJournal.xml and tests whether it matches the number of registry entries found in arkivstruktur.xml`